lib/Net/Easypost/Address.pm

```diff
@@ -162,14 +174,19 @@ sub merge {
 =method verify
 
 This method takes a L<Net::Easypost::Address> object and verifies its underlying
-address
+address.
+
+If a non-US address is asked for verification, a warning will be
+emitted and the object itself will be returned.
 
 =cut
 
 sub verify {
     my $self = shift;
-    use Data::Dumper;
-
+    if ($self->country ne 'US') {
```

Comment:
> Nope, I was missing something. I thought that a prerequisite for
> creating a Shipment was a verified to and from address; I was
> mistaken. 

Actually, to get the labels it looks like I'm missing some pieces (I
suspect CustomsInfo, which is mandatory for international shipping), but
at least now it show the rates.

-- 
Marco